Xamarin.PropertyEditing.Mac/Controls/FilePathEditorControl.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections;
+using AppKit;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class FilePathEditorControl : PropertyEditorControl<PropertyViewModel<FilePath>>
+	{
+		const string SelectFileTitle = "Choose a File";
+
+		NSOpenPanel panel;

```

Comment:
On proppy, as part of out code standards, we are moving to explicitly defining scope. So please add private fields like this. Some of my old code doesn't do this, but every time I go back I update it to the standard.